features/js-modules-shared-workers.yml.dist

```diff
@@ -8,7 +8,7 @@ status:
     edge: "80"
     firefox: "114"
     firefox_android: "114"
-    safari: "15"
-    safari_ios: "15"
+    safari: "16"
```

Comment:
The 16 comes from https://github.com/mdn/browser-compat-data/blob/ebb58c142ff3f1dcade8857de96d16fb8e2f80f1/api/SharedWorker.json#L35.

It seems like this shouldn't pass BCD's lints, there claim that ES modules is supported in Safari 15 is at odds with the data for `SharedWorker`.

Reply:
I filed https://github.com/mdn/browser-compat-data/issues/23624

Reply:
I see. I've tried to figure out if TypeScript can constrain strings to a specific format but don't know the answer. How about we just make `yamlEntries` throw an error if an identifier isn't in the form we like?